docs/data-sources/connection.md

```diff
@@ -29,8 +29,6 @@ Data source to retrieve a specific Auth0 connection by `connection_id` or `name`
 - `realms` (List of String) Defines the realms for which the connection will be used (e.g., email domains). If not specified, the connection name is added as the realm.
 - `show_as_button` (Boolean) Display connection as a button. Only available on enterprise connections.
 - `strategy` (String) Type of the connection, which indicates the identity provider.
-- `strategy_version` (String)
```

Comment:
These 2 properties shouldn't have been neither in the data source or the resource connection docs. These are part of the current state upgraders only and are old remnants of past provider versions.